src/main/sensors/compass.c

```diff
@@ -58,7 +58,7 @@ mag_t mag;                   // mag access functions
 #define COMPASS_INTERRUPT_TAG   IO_TAG_NONE
 #endif
 
-PG_REGISTER_WITH_RESET_FN(compassConfig_t, compassConfig, PG_COMPASS_CONFIG, 0);
+PG_REGISTER_WITH_RESET_FN(compassConfig_t, compassConfig, PG_COMPASS_CONFIG, 1);
```

Comment:
Not sure if this is needed. `compassConfig_t` does not seem to be modified in this pull request, so incrementing the version should not be needed - or I am overlooking something.

Reply:
In the original PR they do it in iNav to force to perform the calibration again to the just because the actual values are not valid (I don't know if for Betaflight is the same).

Reply:
I see. Since we are not using the PG version to detect and reset a parameter group (and since resetting them groupwise is problematic in my opinion anyway), this won't help for this. The only thing that would help is to increase the EEPROM config version, which will be happening between 3.3 and 3.4 anyway.

Reply:
PG version reset should work fine.

Reply:
@ledvinap: Does it actually do anything in Betaflight yet? I thought it didn't (and the bug reports we are getting from users with invalid configuration suggest so too).